Detections/AWSCloudTrail/AWS_CredentialHijack.yaml

```diff
@@ -20,9 +21,9 @@ relevantTechniques:
   - T1087
 query: |
 
-  let timeframe = 1d;
+  let timeframe = 1h;
   AWSCloudTrail
-  | where TimeGenerated >= ago(timeframe)
+  | where TimeGenerated > ago(timeframe)
   | where EventName == "GetCallerIdentity"
   | where UserIdentityType == "AssumedRole" 
   | summarize by SourceIpAddress, TimeGenerated, EventName, EventTypeName, UserIdentityType, UserIdentityAccountId, UserIdentityPrincipalid, 
```

Comment:
Can we aggregate on min() and max() for TimeGenerated and any others to reduce the resulting rowset?